compatibility/bazel_tools/daml_sdk.bzl

```diff
@@ -68,6 +68,10 @@ def _daml_sdk_impl(ctx):
     ctx.extract(
         "ledger-api-test-tool.jar",
         output = "extracted-test-tool",
+        # We cannot fully extract the JAR because there are files
+        # that clash on case insensitive file systems. Luckily, we only
+        # need the DAR so this is not an issue.
+        stripPrefix = "ledger/test-common",
```

Comment:
Maybe something worth fixing in ledger-api-test-tool?

Reply:
I took a brief look at it but I couldn’t find a nice way to configure this.The deploy jars built by Bazel just bundle everything and jarjar only allows zapping classes not files in META-INF afaict.